### ISSUE TITLE
Asking for help about siamrpn++ in pysot

### ISSUE THREAD

**Issue by amirhamidihd** [HanlynnKe/TIR-Tracking#1]
I am writing to enquire about the issues in relation to the PYSOT toolkit (in the object tracking field). Having read your comments on one of the issues in Pysot, I found out that you had the same problems in getting matched results between SIAMRPN++ paper and PYSOT toolkit. I was wondering if you could give a hint to so solve the mismatched results. Have you succeeded in coping with this issue. I would be really grateful if you could help me and give me a clue in this regard.

Note: The mismatched results that I am talking about are mainly related to the pre-trained models of the SIAMRPN++ for OTB

**Comment by HanlynnKe** [comment]
Thank you for your issue. 

> Having read your comments on one of the issues in Pysot, I found out that you had the same problems in getting matched results between SIAMRPN++ paper and PYSOT toolkit.

In my memory, the comments I have written is about training the model on my own dataset. My story is:

I trained model of the SiamRPN++ from scratch with the VOT2018 dataset and yes the results were somehow not as the same as the ones mentioned in the paper. But I thought that the results were acceptable because that experiment environment is not the same: in the paper, the authors used distributed training package in PyTorch while I only had one card which means I could only train the model on one machine. The training process was painful and time-consuming. So I switched to my own dataset and VOT2019 dataset, the smaller ones, to train the model again from scratch.

> I was wondering if you could give a hint to so solve the mismatched results.

If the difference between the performance data from your machine and the one from paper is acceptable, I think this is a normal scene. I have read the backbone codes and I think the structure of the backbone in paper is the same as that in codes. Sorry that I did not tried the pre-trained model on OTB.
